study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/EventBuilder.java

```diff
@@ -258,7 +258,10 @@ private long insertEventAction(Handle handle, Config actionCfg, Config triggerCf
                     .orElseThrow(() -> new DDPException("Could not find pdf configuration with name " + pdfName));
             return actionDao.insertPdfGenerationAction(pdfId);
         } else if (EventActionType.ACTIVITY_INSTANCE_CREATION.name().equals(type)) {
-            String activityCode = actionCfg.getString(ACTIVITY_CODE_FIELD);
+            String activityCode = actionCfg.getString("ACTIVITY_CODE_FIELD");
+            Boolean createFromAnswer = ConfigUtil.getBoolIfPresent(actionCfg, "createFromAnswer");
```

Comment:
Just to be extra careful, can we add a check that, if `createFromAnswer` is true then `triggerCfg` type needs to be `ACTIVITY_STATUS`?

Reply:
Fixed